api/src/wfl/environment.clj

```diff
@@ -52,7 +52,7 @@
    "WFL_CROMWELL_URL"
    #(-> "https://cromwell-gotc-auth.gotc-dev.broadinstitute.org")
    "WFL_TDR_DEFAULT_PROFILE"
-   #(-> "390e7a85-d47f-4531-b612-165fc977d3bd")
+   #(-> "6370f5a1-d777-4991-8200-ceab83521d43")
    "WFL_TDR_SA"
    #(-> "jade-k8-sa@broad-jade-dev.iam.gserviceaccount.com")
```

Comment:
The service account does not change?

Reply:
Nice catch! I actually meant to remove it since it's not used anywhere anymore, somehow I forgot to commit that change…